starter-code/app.js

```diff
@@ -1,5 +1,61 @@
 const express = require('express');
 const app = express();
+const expressLayouts = require('express-ejs-layouts');
 const Chuck  = require('chucknorris-io');
 const client = new Chuck();
 
+// -- setup
+
+app.set('views', __dirname + '/views');
+app.set('view engine', 'ejs');
+
+// -- routes
+
+app.get('/random', (req, res, next) => {
+        // Retrieve a random chuck joke
+        next("SDFAFSDFSDFSDF");
+    client.getRandomJoke()
+    .then((response) => {
+        res.render('index',response);
```

Comment:
You want to pass an object to the render method. There are several ways to do that. You can define that variable before or you can write:
```
res.render('index', {joke: joke.value});
```
In this case, the first "joke" is the name of the property (you can call it whatever you want). The second "joke" is the response of getRandomJoke (remember you were calling it "response" before, but we change its name in the previous comment). Value is a property of that response and the only thing you need to pass to the render method.